src/Microsoft.Diagnostics.Runtime/src/Common/ClrArray.cs

```diff
@@ -199,7 +199,7 @@ private ulong GetItemAddress(int index, bool isObjectReference)
             }
         }
 
-        public ClrValueType GetStructValue(int index)
+        public ClrValueType GetStructValue(int index, bool interior = false)
```

Comment:
This is technically a breaking change.  I think we'll need to structure it this way to preserve binary compat:

```suggestion
        public ClrValueType GetStructValue(int index) => GetStructValue(index, false);

        public ClrValueType GetStructValue(int index, bool interior)
```

Reply:
@chrisnas This change LGTM, if you wouldn't mind committing the above change.

Thanks!

Reply:
I hesitated between the two solutions when I wrote it  :^)
I will do the change tomorrow